apps/server/src/migrations/mikro-orm/Migration20240528140356.ts

```diff
@@ -0,0 +1,31 @@
+import { Migration } from '@mikro-orm/migrations-mongodb';
+
+export class Migration20240528140356 extends Migration {
+	async up(): Promise<void> {
+		const mediaBoards = await this.driver.nativeUpdate(
+			'boardnodes',
+			{ type: 'media-board' },
+			{
+				$rename: { mediaAvailableLineBackgroundColor: 'backgroundColor', mediaAvailableLineCollapsed: 'collapsed' },
```

Comment:
stems from moving to traits, removing duplicate code by reusing common props etc
it would be a problem if media board needs to store backgroundColor also for itself, and can be changed if needed. But for now, it makes the code simpler